src/python/pants/util/contextutil.py

```diff
@@ -98,6 +99,15 @@ def stdio_as(stdout_fd, stderr_fd, stdin_fd):
     yield
 
 
+@contextmanager
+def signal_handler_as(sig, handler):
```

Comment:
It would be great to keep-up doc discipline in util modules in-particular.

Reply:
yep, my bad - fixed.

Reply:
+1